cmd/katana/main.go

```diff
@@ -68,13 +69,62 @@ func main() {
 		gologger.Fatal().Msgf("could not execute crawling: %s", err)
 	}
 
-	// on successful execution remove the resume file in case it exists
+	// on successful execution:
+
+	// deduplicate the lines in each file in the store-field-dir
+	//use options.StoreFieldDir once https://github.com/projectdiscovery/katana/pull/877 is merged
+	storeFieldDir := "katana_field"
+	_ = deduplicateLinesInFilesInDir(storeFieldDir)
+
+	// remove the resume file in case it exists
 	if fileutil.FileExists(resumeFilename) {
 		os.Remove(resumeFilename)
 	}
 
 }
 
+func deduplicateLinesInFilesInDir(dir string) error {
```

Comment:
What about moving them in utils with a note about potential high memory with large files (in our case the output of katana should never big that huge)